### ISSUE TITLE
Optional Sidebar?

### ISSUE THREAD

**Issue by sbrl** [sbrl/Pepperminty-Wiki#7]
Since this wiki software is aimed at small-ish wikis, perhaps having an optional sidebar that shows a list of all the pages & folders on the current wiki would be a good idea.

Quote from [/u/Theis](//reddit.com/u/Theis):

> A better navigation system would be useful. Personal I would like to use this for note taking. So a sidebare with folders and markdown pages to navigate would be beautiful :)

The folders depend on issue #6 though. Things to do:

 - [x] Implement issue #6 
 - [x] Add support for HTML insertion by modules
 - [x] Implement sidebar
 - [x] Add option to settings to turn the sidebar off / on

**Comment by sbrl** [comment]
Link to original reddit post: http://www.reddit.com/r/wiki/comments/37b003/pepperminty_wiki_a_markdownpowered_wiki_in_a/

**Comment by sbrl** [comment]
Sidebar: implemented (commit coming soon). Next up: applying CSS to said sidebar.

**Comment by sbrl** [comment]
The sidebar is themed!
Here's what it looks like currently:
![image](https://cloud.githubusercontent.com/assets/9929737/9069904/12acfad6-3ae4-11e5-8ec4-6ec0e3de7249.png)

Suggestions are welcome, open a new issue or tweet @SBRLabs
